pymeasure/instruments/keithley/keithley2200.py

```diff
@@ -0,0 +1,115 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+from pymeasure.instruments import Instrument, Channel
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class PSChannel(Channel):
+    """ Implementation of a Keithley 2200 channel. """
+
+    VOLTAGE_RANGE = [0, 70]
+    CURRENT_RANGE = [0, 45]
+
+    output_enabled = Instrument.control(
+        "SOURCE:OUTP:ENAB?", "SOURCE:OUTP:ENAB %d",
+        """ A boolean property representing output state.""",
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True
+    )
+
+    voltage = Instrument.control(
```

Comment:
We have started to standardise naming of power supply properties, could you adapt your properties?
https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#power-supplies (`voltage` and `current` as measurements (i.e. get-only), and `voltage_setpoint` and `current_limit` as settings or controls (depending if you can read those back from this device).

Reply:
The voltage you set (this property) should be called `voltage_setpoint`.

What you call "measure_voltage" should be the actually measured voltage, as @bilderbuchi pointed out.

Please don't resolve a conversation, if you did not implement the requested change.